### PR TITLE
Wallet Button (Metamask & WalletConnect) States

### PR DESCRIPTION
**Pull request by math-marcellino**
1. Add Tailwind Colors Configuration
2. Add Hover State to Wallet Button 
3. Add Pressed State to Wallet Button
4. Fix Loading State in Wallet Button

Note: Detailed explanation is in the commits description.

### CODE REVIEW

File: components/v1/Buttons/ConnectWalletDesktop.tsx [hunk 1]
```diff
@@ -131,7 +131,8 @@ const ButtonConnectWalletDesktop: FunctionComponent<ButtonConnectWalletDesktopPr
 
                         <div className="flex flex-col p-4 space-y-2">
                             <button
-                                className="bg-orange-light-2 dark:bg-orange-dark-2 border border-orange-light-5 dark:border-orange-dark-5 rounded-[12px] py-[11px] px-[12px] text-left w-full m-0 flex flex-row items-center justify-between"
+                                className={`bg-orange-light-2 dark:bg-orange-dark-2 border border-orange-light-5 dark:border-orange-dark-5 rounded-[12px] py-[11px] px-[12px] text-left w-full m-0 flex flex-row items-center justify-between transition ease-in-out duration-300 active:scale-95 hover:bg-orange-light-3 dark:hover:bg-orange-dark-3 ${isConnecting && connectorName ? "cursor-wait" : "cursor-pointer"}`}
+                                disabled={isConnecting && connectorName ? true : false}
```
Comment (pyk) [inline components/v1/Buttons/ConnectWalletDesktop.tsx:135]: @math-marcellino ini `connectorName` perlu di cek valuenya juga harusnya ya?

`connectorName === 'metamask'` ?

File: components/v1/Buttons/ConnectWalletDesktop.tsx [hunk 2]
```diff
@@ -151,7 +152,8 @@ const ButtonConnectWalletDesktop: FunctionComponent<ButtonConnectWalletDesktopPr
                                 )}
                             </button>
                             <button
-                                className="bg-blue-light-2 dark:bg-blue-dark-2 border border-blue-light-5 dark:border-blue-dark-5 rounded-[12px] py-[11px] px-[12px] text-left w-full m-0 flex flex-row items-center justify-between"
+                                className={`bg-blue-light-2 dark:bg-blue-dark-2 border border-blue-light-5 dark:border-blue-dark-5 rounded-[12px] py-[11px] px-[12px] text-left w-full m-0 flex flex-row items-center justify-between transition ease-in-out duration-300 active:scale-95 hover:bg-blue-light-3 dark:hover:bg-blue-dark-3 ${isConnecting && connectorName ? "cursor-wait" : "cursor-pointer"}`}
+                                disabled={isConnecting && connectorName ? true : false}
```
Comment (pyk) [inline components/v1/Buttons/ConnectWalletDesktop.tsx:156]: This one too

File: components/v1/Buttons/ConnectWalletMobile.tsx [hunk 3]
```diff
@@ -282,7 +283,8 @@ const ButtonConnectWalletMobile: FunctionComponent<ButtonConnectWalletMobileProp
                                                                 )}
                                                             </button>
                                                             <button
-                                                                className="bg-blue-light-2 dark:bg-blue-dark-2 border border-blue-light-5 dark:border-blue-dark-5 rounded-[12px] py-[11px] px-[12px] text-left m-0 flex flex-row items-center justify-between"
+                                                                className={`bg-blue-light-2 dark:bg-blue-dark-2 border border-blue-light-5 dark:border-blue-dark-5 rounded-[12px] py-[11px] px-[12px] text-left m-0 flex flex-row items-center justify-between transition ease-in-out duration-300 active:scale-95 hover:bg-blue-light-3 dark:hover:bg-blue-dark-3 ${isConnecting && connectorName ? "cursor-wait" : "cursor-pointer"}`}
+                                                                disabled={isConnecting && connectorName ? true : false}
```
Comment (pyk) [inline components/v1/Buttons/ConnectWalletMobile.tsx:287]: I think we need to check "conenctorName == walletconnect" to make sure

**Review by pyk — CHANGES_REQUESTED** [review]

File: components/v1/Buttons/ConnectWalletDesktop.tsx (hunk 1 above)
Comment (math-marcellino) [reply components/v1/Buttons/ConnectWalletDesktop.tsx:135]: Oh kalau itu sebenernya aku sengaja mas, supaya kalau misalnya lagi connecting ke metamask, yang tombol walletconnectnya juga gabisa ditekan. Tapi kalau menurut mas lebih baik gimana?

Comment (pyk) [reply components/v1/Buttons/ConnectWalletDesktop.tsx:135]: @math-marcellino ohh ok, make sense 👍

**Review by pyk — APPROVED** [review]

**Comment by pyk** [comment]
LGTM

**Pull request merged**